src/components/common/Layout/components/Account/components/Notifications/MessageSubscriptionModal.tsx

```diff
@@ -0,0 +1,100 @@
+import {
```

Comment:
I think we don't use this component, right? Let's delete it if we don't need it!

Reply:
true